<launch>
    <node name="waypoint_follower_node" pkg="waypoint_follower"
        type="waypoint_follower_node" output="screen">
        <param name="max_waypoints" type="int" value="5"/>
        <param name="loop_rate" type="double" value="10.0"/>
        <param name="wait_timeout" type="double" value="10.0"/>
        <param name="move_base_server_name" type="string" value="move_base"/>
        <param name="waypoint_topic" type="string" value="waypoints"/>
        <param name="move_base_frame_id" type="string" value="/map"/>
    </node>


</launch>